### PR TITLE
Extend launcher to generate logging.json.

### PR DESCRIPTION
**Pull request by jgiszczak**
Revise later to deploy to data directory when eosd supports it.

Does what it says on the tin.

### CODE REVIEW

File: programs/launcher/main.cpp [hunk 1]
```diff
@@ -353,8 +354,11 @@ struct launcher_def {
   void define_network ();
   void bind_nodes ();
   host_def *find_host (const string &name);
-  host_def *deploy_config_file (tn_node_def &node);
+  host_def *deploy_config_files (tn_node_def &node);
+  string compose_scp_command (const host_def &host, const bf::path &source,
+                              const bf::path &destination);
```
Comment (heifner) [inline programs/launcher/main.cpp]: At some point need to clean up these files so they follow the style of the rest of the code base. e.g. `const host_def& host, const bf::path& source, const bf::path& destination`

File: programs/launcher/main.cpp [hunk 2]
```diff
@@ -840,6 +866,38 @@ launcher_def::write_config_file (tn_node_def &node) {
...
+  if (!cfg.good()) {
+    cerr << "unable to open " << filename << " " << strerror(errno) << "\n";
+    exit (9);
+  }
+
+  auto log_config = fc::logging_config::default_config();
+  log_config.appenders.push_back(
+        fc::appender_config( "net", "gelf",
+            fc::mutable_variant_object()
+                ( "endpoint", "10.160.11.21:12201" )
```
Comment (heifner) [inline programs/launcher/main.cpp]: Hardcoded? This needs to be configurable.

Comment (jgiszczak) [reply programs/launcher/main.cpp]: Next version.  There's only one gelf server today.

Comment (heifner) [reply programs/launcher/main.cpp]: Hardcoded value will be addressed by: STAT-268

**Pull request merged**